packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -269,9 +270,7 @@ export default class DataGrid extends React.Component {
 			`.${FOCUSED_COLUMN_CLASS_NAME}`,
 		);
 
-		for (const focusedCell of focusedCells) {
-			focusedCell.classList.remove(FOCUSED_COLUMN_CLASS_NAME);
-		}
+		focusedCells.forEach(({ classList }) => classList.remove(FOCUSED_COLUMN_CLASS_NAME));
```

Comment:
We need to check that IE has a polyfill for this method as it doesn't support forEach on NodeList. See: https://developer.mozilla.org/fr/docs/Web/API/NodeList/forEach

Reply:
we use babel polyfill to add this method. https://github.com/Talend/ui-scripts/blob/master/packages/config-babel/.browserslistrc